sdk/trace/trace_test.go

```diff
@@ -1605,6 +1605,70 @@ func TestAddLinksWithMoreAttributesThanLimit(t *testing.T) {
 	}
 }
 
+type FrozenClock struct {
```

Comment:
```suggestion
type frozenClock struct {
```

This (and the other test types) doesn't need to be exported, does it?